instrumentation/cassandra-datastax-4.0.0/src/test/java/com/nr/agent/instrumentation/cassandra/CassandraNoInstrumentation.java

```diff
@@ -32,8 +33,8 @@
 import static org.junit.Assert.assertEquals;
 
 // Issue when running cassandra unit on Java 9+ - https://github.com/jsevellec/cassandra-unit/issues/249
-@Category({Java9IncompatibleTest.class, Java10IncompatibleTest.class, Java11IncompatibleTest.class, Java12IncompatibleTest.class, Java13IncompatibleTest.class, Java14IncompatibleTest.class,
-        Java15IncompatibleTest.class, Java16IncompatibleTest.class, Java17IncompatibleTest.class, Java18IncompatibleTest.class})
+@Category({ Java9IncompatibleTest.class, Java10IncompatibleTest.class, Java11IncompatibleTest.class, Java12IncompatibleTest.class, Java13IncompatibleTest.class, Java14IncompatibleTest.class,
+        Java15IncompatibleTest.class, Java16IncompatibleTest.class, Java17IncompatibleTest.class, Java18IncompatibleTest.class, Java19IncompatibleTest.class })
 @RunWith(InstrumentationTestRunner.class)
```

Comment:
I've been wondering about this growing list of annotations similar to what you called out on the long list of `project.hasProperty(...)` checks in some build.gradles.  Not for this PR, but do you think we should consider a long term solution for these annotations as well?

Reply:
Honestly, we should just update these to only have annotations for Java versions that we actually test against. 

We should be able to simplify it to just this:

```
@Category({ Java11IncompatibleTest.class, Java17IncompatibleTest.class, Java19IncompatibleTest.class })
```

Reply:
I think I'll leave such a change for a different PR, so we can discuss first.